Fix non-reproducibilities caused by time-dependent procedures.
Submitted upstream: https://github.com/ipython/ipython/pull/13640

diff --git a/IPython/sphinxext/ipython_directive.py b/IPython/sphinxext/ipython_directive.py
index 18bdfcae9..2c665ac87 100644
--- a/IPython/sphinxext/ipython_directive.py
+++ b/IPython/sphinxext/ipython_directive.py
@@ -19,7 +19,7 @@
    In [1]: 1+1
 
    In [1]: import datetime
-      ...: datetime.datetime.now()
+      ...: datetime.date.fromisoformat('2022-02-22')
 
 It supports IPython construct that plain
 Python does not understand (like magics):
@@ -28,7 +28,7 @@
 
    In [0]: import time
 
-   In [0]: %timeit time.sleep(0.05)
+   In [0]: %pdoc time
 
 This will also support top-level async when using IPython 7.0+
 
